packages/mui-styles/src/createGenerateClassName/createGenerateClassName.js

```diff
@@ -10,13 +10,15 @@ import { unstable_nested as nested } from '@mui/private-theming/ThemeProvider';
  * being untouched by simple user overrides.
  */
 const stateClasses = [
+  'active',
   'checked',
+  'completed',
   'disabled',
   'error',
+  'expanded',
   'focused',
   'focusVisible',
   'required',
-  'expanded',
   'selected',
 ];
```

Comment:
This seems to be a breaking change. We are modifying @mui/styles here. Instead, could we make it so that the docs rely on `/packages/mui-base/src/generateUtilityClass/generateUtilityClass.ts` for the generation of the docs, to match what the implement uses.

Reply:
Good point, thanks. I've changed it.